### PR TITLE
Fixed width issue in snapshot testing for Areachart, VerticalbarChart, HeatmapChart, HorizontalbarchartWithAxis, Linechart and GroupedVerticalStackedBarChart

### PR DESCRIPTION
**Pull request by v-baambati**
Fixed snapshot width issue for below charts,
Areachart,
VerticalbarChart,
HeatmapChart, 
HorizontalbarchartWithAxis, 
Linechart
GroupedVerticalStackedBarChart

### CODE REVIEW

**Comment by srmukher** [comment]
Please update the description.

File: packages/react-charting/src/components/AreaChart/AreaChartRTL.test.tsx [hunk 1]
```diff
@@ -252,7 +275,12 @@ const chartDataWithDates = {
 };
 
 describe('Area chart rendering', () => {
-  beforeEach(sharedBeforeEach);
+  beforeEach(() => {
+    sharedBeforeEach();
+    updateChartWidthAndHeight();
+  });
+  afterEach(sharedAfterEach);
```
Comment (srmukher) [inline packages/react-charting/src/components/AreaChart/AreaChartRTL.test.tsx:282]: Similarly for afterEach which is repetitive

File: packages/react-charting/src/components/GroupedVerticalBarChart/GroupedVerticalBarChartRTL.test.tsx [hunk 2]
```diff
@@ -117,7 +140,12 @@ const chartPoints = [
 ];
 
 describe('Grouped Vertical bar chart rendering', () => {
-  beforeEach(sharedBeforeEach);
+  beforeEach(() => {
```
Comment (srmukher) [inline packages/react-charting/src/components/GroupedVerticalBarChart/GroupedVerticalBarChartRTL.test.tsx:143]: Similar to AreaChart, make all repetitive beforeEach/afterEach global

**Comment by srmukher** [comment]
@v-baambati As commented in AreaChart, make all repetitive beforeEach/afterEach global in every test file.

**Comment by fabricteam** [comment]
<div id="bundleSizeComment9423"/>

File: packages/react-charting/UnitTests/VerticalBarChartUT.test.tsx [hunk 3]
```diff
@@ -25,13 +25,11 @@ const stringPoints = [
   },
```
Comment (fabricteam) [inline packages/react-charting/UnitTests/VerticalBarChartUT.test.tsx:25]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30771&commitId=7c5a24a5012633d9e37805b25cafcb97ff749d04&env=prod&clientType=FLUENTUI)


  <details><summary><b>react-charting-AreaChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-AreaChart.Custom Accessibility.default.chromium.png|11| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Review by srmukher — APPROVED** [review]

**Review by AtishayMsft — APPROVED** [review]

File: packages/react-charting/UnitTests/VerticalBarChartUT.test.tsx (hunk 3 above)
Comment (AtishayMsft) [reply packages/react-charting/UnitTests/VerticalBarChartUT.test.tsx:25]: VR link is not working. Resolving the comment as the change is only in test files and not in the product.

**Pull request merged**